storagenode/storagenodedb/vouchers.go

```diff
@@ -56,35 +56,27 @@ func (db *vouchersdb) Put(ctx context.Context, voucher *pb.Voucher) (err error)
 	return err
 }
 
-// GetExpiring retrieves all vouchers that are expired or about to expire
-func (db *vouchersdb) GetExpiring(ctx context.Context, expirationBuffer time.Duration) (satellites []storj.NodeID, err error) {
+// NeedVoucher returns true if a satellite's voucher is expired/about to expire or does not exist
```

Comment:
super nit: "NeedVoucher returns true if a voucher from a particular satellite is expired, about to expire, or does not exist"